NCBI Export fails when handled by IRIDA background task

## Describe the bug

Uploading data to NCBI crashes with the following exception:

```
2023-01-20 09:42:26.299 DEBUG 25707 --- [pool-2-thread-2] c.c.b.i.s.export.ExportUploadService     : Creating export xml for submission 1
2023-01-20 09:42:26.361 ERROR 25707 --- [pool-2-thread-2] org.thymeleaf.TemplateEngine             : [THYMELEAF][pool-2-thread-2] Exception processing template "ncb
i": Exception evaluating SpringEL expression: "sample.instrumentModel.value" (template: "ncbi" - line 75, col 39)

org.thymeleaf.exceptions.TemplateProcessingException: Exception evaluating SpringEL expression: "sample.instrumentModel.value" (template: "ncbi" - line 75, col 39)
        at org.thymeleaf.spring5.expression.SPELVariableExpressionEvaluator.evaluate(SPELVariableExpressionEvaluator.java:292) ~[thymeleaf-spring5-3.0.15.RELEASE.ja
r:3.0.15.RELEASE]
        at org.thymeleaf.standard.expression.VariableExpression.executeVariableExpression(VariableExpression.java:166) ~[thymeleaf-3.0.15.RELEASE.jar:3.0.15.RELEASE
]
        at org.thymeleaf.standard.expression.SimpleExpression.executeSimple(SimpleExpression.java:66) ~[thymeleaf-3.0.15.RELEASE.jar:3.0.15.RELEASE]
...
Caused by: org.springframework.expression.spel.SpelEvaluationException: EL1008E: Property or field 'value' cannot be found on object of type 'ca.corefacility.bioinf
ormatics.irida.model.export.NcbiInstrumentModel' - maybe not public or not valid?
        at org.springframework.expression.spel.ast.PropertyOrFieldReference.readProperty(PropertyOrFieldReference.java:217) ~[spring-expression-5.3.23.jar:5.3.23]
...
2023-01-20 09:42:26.362 ERROR 25707 --- [pool-2-thread-2] c.c.b.i.s.export.ExportUploadService     : Upload failed
```

This occurs in the latest stable version (22.09.6).

## Steps to reproduce the problem
What were you doing when you encountered the problem?
1. Select samples and create new NCBI Export. Submit the NCBI Export.
2. The export will appear to be submitted successfully.
3. Wait until the NCBI Export is picked up by the IRIDA background task (this means IRIDA must be run in prod mode or with the NCBI profile enabled).
4. When the NCBI Export is picked up by the background task, it will fail with the above exception.

## Expected behaviour
I expected the NCBI Export to succeed.

## Additional context
None.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 
 # Changelog
 
+## [22.09.7] - Unreleased
+* [UI]: Fixed bugs on NCBI Export page preventing the NCBI `submission.xml` file from being properly written. See [PR 1451](https://github.com/phac-nml/irida/pull/1451)
+
 ## [22.09.6] - 2022/12/21
 * [UI]: Fixed bug on NCBI Export page preventing the export from occuring. See [PR 1439](https://github.com/phac-nml/irida/pull/1439)
 

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -14,7 +14,7 @@ plugins {
 }
 
 group = "ca.corefacility.bioinformatics"
-version = "22.09.6"
+version = "22.09.7"
 description = "irida"
 
 java {

--- a/src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UINcbiService.java
+++ b/src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UINcbiService.java
@@ -14,14 +14,13 @@ import ca.corefacility.bioinformatics.irida.model.NcbiExportSubmission;
 import ca.corefacility.bioinformatics.irida.model.export.NcbiBioSampleFiles;
 import ca.corefacility.bioinformatics.irida.model.project.Project;
 import ca.corefacility.bioinformatics.irida.model.sequenceFile.SequenceFilePair;
-import ca.corefacility.bioinformatics.irida.model.sequenceFile.SequencingObject;
 import ca.corefacility.bioinformatics.irida.model.sequenceFile.SingleEndSequenceFile;
 import ca.corefacility.bioinformatics.irida.model.user.User;
 import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.NcbiExportSubmissionTableModel;
 import ca.corefacility.bioinformatics.irida.ria.web.models.export.NcbiBioSampleModel;
 import ca.corefacility.bioinformatics.irida.ria.web.models.export.NcbiExportSubmissionAdminTableModel;
-import ca.corefacility.bioinformatics.irida.ria.web.models.export.NcbiSubmissionRequest;
 import ca.corefacility.bioinformatics.irida.ria.web.models.export.NcbiSubmissionModel;
+import ca.corefacility.bioinformatics.irida.ria.web.models.export.NcbiSubmissionRequest;
 import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableRequest;
 import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableResponse;
 import ca.corefacility.bioinformatics.irida.service.ProjectService;
@@ -89,22 +88,8 @@ public class UINcbiService {
 	 */
 	public NcbiSubmissionModel getExportDetails(Long exportId) {
 		NcbiExportSubmission submission = ncbiService.read(exportId);
-		Project project = projectService.read(submission.getProject()
-				.getId());
 
-		List<NcbiBioSampleModel> bioSamples = submission.getBioSampleFiles()
-				.stream()
-				.map(bioSampleFile -> {
-					List<SequencingObject> pairs = bioSampleFile.getPairs()
-							.stream()
-							.peek(pair -> uiSampleService.enhanceQcEntries(pair, project))
-							.collect(Collectors.toList());
-					List<SequencingObject> singles = bioSampleFile.getFiles()
-							.stream()
-							.peek(single -> uiSampleService.enhanceQcEntries(single, project))
-							.collect(Collectors.toList());
-					return new NcbiBioSampleModel(bioSampleFile);
-				})
+		List<NcbiBioSampleModel> bioSamples = submission.getBioSampleFiles().stream().map(NcbiBioSampleModel::new)
 				.collect(Collectors.toList());
 
 		return new NcbiSubmissionModel(submission, bioSamples);

--- a/src/main/resources/ca/corefacility/bioinformatics/irida/export/ncbi.xml
+++ b/src/main/resources/ca/corefacility/bioinformatics/irida/export/ncbi.xml
@@ -1,7 +1,8 @@
 <?xml version='1.0' encoding='utf-8'?>
 
 <Submission schema_version="2.0" xmlns:th="http://www.thymeleaf.org"
-	xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:noNamespaceSchemaLocation="http://www.ncbi.nlm.nih.gov/viewvc/v1/trunk/submit/public-docs/common/submission.xsd?view=co">
 	<Description>
 		<Organization role="owner" type="institute">
@@ -17,13 +18,12 @@
 	</Description>
 
 	<Action th:each="sample : ${submission.bioSampleFiles}">
-		<!--/*-->
-		IRIDA sequence files may either be plaintext (.fastq) or gzip compressed (.fastq.gz)
-		NCBI requires that uploaded files should have the appropriate file extension.
-		Determine if the IRIDA sequence file is compressed by looking for .gz extension
-		and add the appropriate extension to the uploaded files.
-		<!--*/-->
-		<AddFiles target_db="SRA" th:each="file : ${sample.files}">
+		<!--/*--> IRIDA sequence files may either be
+		plaintext (.fastq) or gzip compressed (.fastq.gz) NCBI requires that uploaded files should
+		have the appropriate file extension. Determine if the IRIDA sequence file is compressed by
+		looking for .gz extension and add the appropriate extension to the uploaded files. <!--*/-->
+		<AddFiles
+			target_db="SRA" th:each="file : ${sample.files}">
 			<th:block th:if="${file.getFiles().toString().endsWith(&quot;.gz&quot;)}">
 				<File th:attr="file_path=${file.getId() + '.fastq.gz'}">
 					<DataType>generic-data</DataType>
@@ -38,9 +38,11 @@
 			<Attribute name="library_name" th:text="${sample.libraryName}">_name_</Attribute>
 			<Attribute name="library_strategy" th:text="${sample.libraryStrategy.value}">_Strategy_</Attribute>
 			<Attribute name="library_source" th:text="${sample.librarySource.value}">_Source_</Attribute>
-			<Attribute name="library_selection" th:text="${sample.librarySelection.value}">_Selection_</Attribute>
+			<Attribute name="library_selection" th:text="${sample.librarySelection.value}">
+		_Selection_</Attribute>
 			<Attribute name="library_layout">SINGLE</Attribute>
-			<Attribute name="library_construction_protocol" th:text="${sample.libraryConstructionProtocol}">_Protocol_</Attribute>
+			<Attribute name="library_construction_protocol"
+				th:text="${sample.libraryConstructionProtocol}">_Protocol_</Attribute>
 			<AttributeRefId name="BioProject">
 				<RefId>
 					<PrimaryId db="BioProject" th:text="${submission.bioProjectId}">_Project_</PrimaryId>
@@ -56,8 +58,10 @@
 					th:text="${sample.id}">_spuid_</SPUID>
 			</Identifier>
 		</AddFiles>
-		<AddFiles target_db="SRA" th:each="pair : ${sample.pairs}" th:with="forward=${pair.getForwardSequenceFile()}, reverse=${pair.getReverseSequenceFile()}">
-            <th:block th:if="${forward.getFile().toString().endsWith(&quot;.gz&quot;)}">
+		<AddFiles
+			target_db="SRA" th:each="pair : ${sample.pairs}"
+			th:with="forward=${pair.getForwardSequenceFile()}, reverse=${pair.getReverseSequenceFile()}">
+			<th:block th:if="${forward.getFile().toString().endsWith(&quot;.gz&quot;)}">
 				<File th:attr="file_path=${forward.getId() + '.fastq.gz'}">
 					<DataType>generic-data</DataType>
 				</File>
@@ -67,7 +71,7 @@
 					<DataType>generic-data</DataType>
 				</File>
 			</th:block>
-			<th:block th:if="${reverse.getFile().toString().endsWith(&quot;.gz&quot;)}" >
+			<th:block th:if="${reverse.getFile().toString().endsWith(&quot;.gz&quot;)}">
 				<File th:attr="file_path=${reverse.getId() + '.fastq.gz'}">
 					<DataType>generic-data</DataType>
 				</File>
@@ -81,9 +85,11 @@
 			<Attribute name="library_name" th:text="${sample.libraryName}">_name_</Attribute>
 			<Attribute name="library_strategy" th:text="${sample.libraryStrategy.value}">_Strategy_</Attribute>
 			<Attribute name="library_source" th:text="${sample.librarySource.value}">_Source_</Attribute>
-			<Attribute name="library_selection" th:text="${sample.librarySelection.value}">_Selection_</Attribute>
+			<Attribute name="library_selection" th:text="${sample.librarySelection.value}">
+		_Selection_</Attribute>
 			<Attribute name="library_layout">PAIRED</Attribute>
-			<Attribute name="library_construction_protocol" th:text="${sample.libraryConstructionProtocol}">_Protocol_</Attribute>
+			<Attribute name="library_construction_protocol"
+				th:text="${sample.libraryConstructionProtocol}">_Protocol_</Attribute>
 			<AttributeRefId name="BioProject">
 				<RefId>
 					<PrimaryId db="BioProject" th:text="${submission.bioProjectId}">_Project_</PrimaryId>

--- a/src/main/resources/ca/corefacility/bioinformatics/irida/export/ncbi.xml
+++ b/src/main/resources/ca/corefacility/bioinformatics/irida/export/ncbi.xml
@@ -34,7 +34,7 @@
 					<DataType>generic-data</DataType>
 				</File>
 			</th:block>
-			<Attribute name="instrument_model" th:text="${sample.instrumentModel.value}">_Model_</Attribute>
+			<Attribute name="instrument_model" th:text="${sample.instrumentModel.model}">_Model_</Attribute>
 			<Attribute name="library_name" th:text="${sample.libraryName}">_name_</Attribute>
 			<Attribute name="library_strategy" th:text="${sample.libraryStrategy.value}">_Strategy_</Attribute>
 			<Attribute name="library_source" th:text="${sample.librarySource.value}">_Source_</Attribute>
@@ -77,7 +77,7 @@
 					<DataType>generic-data</DataType>
 				</File>
 			</th:block>
-			<Attribute name="instrument_model" th:text="${sample.instrumentModel.value}">_Model_</Attribute>
+			<Attribute name="instrument_model" th:text="${sample.instrumentModel.model}">_Model_</Attribute>
 			<Attribute name="library_name" th:text="${sample.libraryName}">_name_</Attribute>
 			<Attribute name="library_strategy" th:text="${sample.libraryStrategy.value}">_Strategy_</Attribute>
 			<Attribute name="library_source" th:text="${sample.librarySource.value}">_Source_</Attribute>

--- a/src/main/webapp/resources/js/components/ncbi/details/utils.tsx
+++ b/src/main/webapp/resources/js/components/ncbi/details/utils.tsx
@@ -17,8 +17,14 @@ import ExportUploadStateTag from "../ExportUploadStateTag";
 export const formatNcbiSubmissionDetails = (
   submission: Omit<NcbiSubmission, "bioSamples">
 ): BasicListItem[] => {
+  const LOCALE = window.TL?.LANGUAGE_TAG || "en";
   const releaseDate = submission.releaseDate
-    ? formatInternationalizedDateTime(submission.releaseDate)
+    ? new Date(submission.releaseDate).toLocaleDateString(LOCALE, {
+        month: "short",
+        year: "numeric",
+        day: "numeric",
+        timeZone: "UTC", // ignoring timezone
+      })
     : i18n("NcbiExportDetailsView.not-released");
 
   return [

--- a/src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx
+++ b/src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx
@@ -234,7 +234,7 @@ function CreateNcbiExport(): JSX.Element {
           bioProject,
           namespace,
           organization,
-          releaseDate: releaseDate.unix(),
+          releaseDate: releaseDate.valueOf(),
           samples: Object.values(_samples).map(
             ({
               pairs = [],

--- a/src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx
+++ b/src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx
@@ -234,7 +234,7 @@ function CreateNcbiExport(): JSX.Element {
           bioProject,
           namespace,
           organization,
-          releaseDate: releaseDate.valueOf(),
+          releaseDate: releaseDate.endOf("day").valueOf(),
           samples: Object.values(_samples).map(
             ({
               pairs = [],

--- a/src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx
+++ b/src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx
@@ -219,7 +219,7 @@ function CreateNcbiExport(): JSX.Element {
             pairs?: number[];
             singles?: number[];
             bioSample: string;
-            libraryName: { value: string };
+            libraryName: string;
             libraryStrategy: { value: string };
             librarySource: { value: string };
             libraryConstructionProtocol: { value: string };
@@ -251,7 +251,7 @@ function CreateNcbiExport(): JSX.Element {
                 singles,
                 pairs,
                 bioSample,
-                libraryName: libraryName.value,
+                libraryName,
                 libraryStrategy: libraryStrategy.value,
                 librarySource: librarySource.value,
                 libraryConstructionProtocol: libraryConstructionProtocol.value,

--- a/src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/NcbiExportPage.java
+++ b/src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/NcbiExportPage.java
@@ -4,6 +4,7 @@ import java.time.Duration;
 import java.util.List;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
@@ -29,6 +30,9 @@ public class NcbiExportPage extends AbstractPage {
 
 	@FindBy(xpath = "//*[@id=\"namespace\"]")
 	private WebElement namespaceInput;
+
+	@FindBy(xpath = "//*[@id=\"releaseDate\"]")
+	private WebElement releaseDateInput;
 
 	@FindBy(className = "t-defaults-panel")
 	private WebElement defaultsPanel;
@@ -71,6 +75,12 @@ public class NcbiExportPage extends AbstractPage {
 
 	public void enterNamespace(String value) {
 		namespaceInput.sendKeys(value);
+	}
+
+	public void setReleaseDateInput(String value) {
+		releaseDateInput.sendKeys(Keys.chord(Keys.CONTROL, "a"));
+		releaseDateInput.sendKeys(value);
+		releaseDateInput.sendKeys(Keys.ENTER);
 	}
 
 	public void toggleDefaultsPanel() {

--- a/src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/NcbiExportsListingPage.java
+++ b/src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/NcbiExportsListingPage.java
@@ -1,11 +1,15 @@
 package ca.corefacility.bioinformatics.irida.ria.integration.pages.projects;
 
+import java.time.Duration;
 import java.util.List;
 
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.AbstractPage;
 
@@ -24,5 +28,12 @@ public class NcbiExportsListingPage extends AbstractPage {
 
 	public int getNumberOfBioSampleIdsDisplayed() {
 		return biodamplesIds.size();
+	}
+
+	public void gotoSubmissionPage(String bioprojectId) {
+		WebElement link = driver.findElement(By.xpath("//table/tbody//a[text() = '" + bioprojectId + "']"));
+		link.click();
+		WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+		wait.until(ExpectedConditions.urlMatches("/projects/\\d+/export/\\d+"));
 	}
 }

--- a/src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/NcbiExportPageIT.java
+++ b/src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/NcbiExportPageIT.java
@@ -4,7 +4,9 @@ import org.junit.jupiter.api.Test;
 
 import ca.corefacility.bioinformatics.irida.ria.integration.AbstractIridaUIITChromeDriver;
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.LoginPage;
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.projects.ExportDetailsPage;
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.projects.NcbiExportPage;
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.projects.NcbiExportsListingPage;
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.projects.ProjectSamplesPage;
 
 import com.github.springtestdbunit.annotation.DatabaseSetup;
@@ -22,6 +24,7 @@ class NcbiExportPageIT extends AbstractIridaUIITChromeDriver {
 		String SAMPLE_3 = "sample3";
 		String BIOPROJECT = "BIOPROJECT-1";
 		String NAMESPACE = "NAMESPACE-FOOBAR";
+		String RELEASE_DATE = "2030-06-15";
 		String ORGANIZATION = "ORGANIZATION-FOOBAR";
 		String PROTOCOL = "AMAZING_PROTOCOL";
 		String DEFAULT_PROTOCOL = "DEFAULT_PROTOCOL";
@@ -40,6 +43,7 @@ class NcbiExportPageIT extends AbstractIridaUIITChromeDriver {
 		page.enterBioProject(BIOPROJECT);
 		page.enterNamespace(NAMESPACE);
 		page.enterOrganization(ORGANIZATION);
+		page.setReleaseDateInput(RELEASE_DATE);
 
 		// Test default sample settings.
 		page.toggleDefaultsPanel();
@@ -84,5 +88,16 @@ class NcbiExportPageIT extends AbstractIridaUIITChromeDriver {
 		assertTrue(page.isSuccessAlertDisplayed(), "Success notification should be displayed");
 		assertTrue(page.isUserRedirectedToProjectSamplesPage(PROJECT_ID),
 				"User should be redirected within 5 seconds of submission");
+
+		// Make sure all fields submitted successfully
+		NcbiExportsListingPage listingPage = NcbiExportsListingPage.goTo(driver());
+		listingPage.gotoSubmissionPage(BIOPROJECT);
+
+		ExportDetailsPage detailsPage = ExportDetailsPage.initPage(driver());
+		assertEquals(BIOPROJECT, detailsPage.getBioproject());
+		assertEquals(NAMESPACE, detailsPage.getNamespace());
+
+		assertEquals(ORGANIZATION, detailsPage.getOrganization());
+		assertTrue(detailsPage.getReleaseDate().equals("Jun 15, 2030"));
 	}
 }

--- a/src/test/java/ca/corefacility/bioinformatics/irida/service/export/ExportUploadServiceTest.java
+++ b/src/test/java/ca/corefacility/bioinformatics/irida/service/export/ExportUploadServiceTest.java
@@ -71,6 +71,7 @@ public class ExportUploadServiceTest {
 				.withTest(xmlContent)
 				.ignoreComments()
 				.ignoreWhitespace()
+				.withNodeFilter(node -> !(node.getNodeName().equals("Hold")))
 				.build();
 		assertFalse(xmlDiff.hasDifferences(), "the resulting xml is not correct");
 	}
@@ -428,7 +429,7 @@ public class ExportUploadServiceTest {
 		ncbiBioSampleFiles.setId("a7f1d71e-8f86-4f0a-8f69-f79eca567f9d");
 
 		NcbiExportSubmission submission = new NcbiExportSubmission(null, submitter, "bioProjectId", "organization",
-				"ncbiNamespace", new Date(1655389918L), Lists.newArrayList(ncbiBioSampleFiles));
+				"ncbiNamespace", new Date(), Lists.newArrayList(ncbiBioSampleFiles));
 		submission.setId(1L);
 
 		return submission;

--- a/src/test/java/ca/corefacility/bioinformatics/irida/service/export/ExportUploadServiceTest.java
+++ b/src/test/java/ca/corefacility/bioinformatics/irida/service/export/ExportUploadServiceTest.java
@@ -18,6 +18,8 @@ import org.mockito.ArgumentCaptor;
 import org.thymeleaf.spring5.SpringTemplateEngine;
 import org.thymeleaf.templatemode.TemplateMode;
 import org.thymeleaf.templateresolver.ClassLoaderTemplateResolver;
+import org.xmlunit.builder.DiffBuilder;
+import org.xmlunit.diff.Diff;
 
 import ca.corefacility.bioinformatics.irida.exceptions.UploadException;
 import ca.corefacility.bioinformatics.irida.model.NcbiExportSubmission;
@@ -64,8 +66,13 @@ public class ExportUploadServiceTest {
 
 		String xmlContent = assertDoesNotThrow(() -> exportUploadService.createXml(submission),
 				"createXml should not raise an exception with valid input.");
-		assertEquals(FileUtils.readFileToString(TEST_NCBI_XML_PATH.toFile(), "UTF-8"), xmlContent,
-				"the xml produced should match");
+		String expectedXmlContent = FileUtils.readFileToString(TEST_NCBI_XML_PATH.toFile(), "UTF-8");
+		Diff xmlDiff = DiffBuilder.compare(expectedXmlContent)
+				.withTest(xmlContent)
+				.ignoreComments()
+				.ignoreWhitespace()
+				.build();
+		assertFalse(xmlDiff.hasDifferences(), "the resulting xml is not correct");
 	}
 
 	@Test

--- a/src/test/resources/files/ncbi-export-test/example-ncbi.xml
+++ b/src/test/resources/files/ncbi-export-test/example-ncbi.xml
@@ -1,6 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 
-<Submission schema_version="2.0" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<Submission schema_version="2.0" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:noNamespaceSchemaLocation="http://www.ncbi.nlm.nih.gov/viewvc/v1/trunk/submit/public-docs/common/submission.xsd?view=co">
 	<Description>
 		<Organization role="owner" type="institute">
@@ -17,7 +18,8 @@
 
 	<Action>
 		
-		<AddFiles target_db="SRA">
+		<AddFiles
+			target_db="SRA">
 			
 			
 				<File file_path="1.fastq">

